src/client/clientmap.cpp

```diff
@@ -124,7 +124,10 @@ void ClientMap::updateDrawList()
 
 	const v3f camera_position = m_camera_position;
 	const v3f camera_direction = m_camera_direction;
-	const f32 camera_fov = m_camera_fov;
+
+	// Use a higher fov to accomodate faster camera movements.
+	// Blocks are cropped better when they are drawn.
+	const f32 camera_fov = m_camera_fov * 1.1;
```

Comment:
Please could this be
`* 1.1f;`
?
Our coding practice is to state floats with an `f` even though they are still not in many places.

Reply:
```suggestion
	const f32 camera_fov = m_camera_fov * 1.1f;
```